jamaljsr/polar
# feat(lnd): add invoice macaroon to Connect tab

### Description

Adds the LND invoice macaroon to the Connect tab.

### Steps to Test

1. Create a new Network with an LND node
1. Start the network
1. Click on the LND node
1. Click on the Connect tab in the sidebar
1. Confirm the invoice macaroon info is listed

### Screenshots

![image](https://user-images.githubusercontent.com/1356600/91925674-81385b00-eca3-11ea-8712-ccdc8c7c35c1.png)
